app/src/main/AndroidManifest.xml

```diff
@@ -14,9 +14,8 @@
   ~ limitations under the License.
   -->
 
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.leinardi.forlago">
-
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```

Comment:
This feels very wrong: we don't need this permission on release builds since leak canary is only needed for debug.

Reply:
The issue was due to not using the latest version of leak canary. Many deps were actually not updated, which is strange since this was the main purpose of this task.